src/jsonld/image.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
```

Comment:
I have started to add Jest coverage for JSON-LD components if you wanted to add one for this component. 

But happy to accept PR without as adding them is a WIP right now. 

Example: https://github.com/garmeeh/next-seo/blob/master/src/jsonld/__tests__/dataset.spec.tsx

Reply:
Cool I'll give that a go!